community/server/src/docs/man/neo4j.1.asciidoc

```diff
@@ -36,6 +37,14 @@ For information regarding Windows, see http://neo4j.com/docs/stable/powershell.h
 *status*::
   Current running state of the server.
 
+.Windows specific commands
```

Comment:
I consider this a compound modifier which should be hyphenated: "Windows-specific commands"